src/styles/index.css

```diff
@@ -1,6 +1,10 @@
 /* stylelint-disable */
 /* eslint-disable */
```

Comment:
- [ ] Instead of disabling `eslint`, please kindly fix the issue since `linter` are there to help you implement best practices.